### DISCUSSION TITLE
How to add nginx and let's encrypt to k8s operator console and tenant  console created by using operator

### DISCUSSION THREAD

**Discussion by meghnaghadge** [minio/minio#16687] [Q&A]
Hi, 
I am using minio k8s operator v4.5.8 to setup production level minio. 
Also I am creating tenant by using below command: 
`kubectl minio tenant create minio-tenant --namespace minio-tenant --storage-class standard --capacity 100Gi --servers 2 --volumes 4`

Now I wanted to use k8s nginx to add my custom domain also need ssl certificate for same domain.
How can I use k8s nginx and let's encrypt to create ssl certificate for my custom domain?

**Comment by harshavardhana** [comment]
Production deployments are advised to get support subscription, feel free to reach out to us to get a subscription for your deployment @meghnaghadge https://min.io/pricing

**Reply by meghnaghadge** [reply]
but even though If I want domain for development env, how can I add using minio operator ?
